include/UIKit/UIViewController.h

```diff
@@ -117,6 +117,7 @@ UIKIT_EXPORT_CLASS
 @property (readonly, nonatomic, strong) NSExtensionContext* extensionContext STUB_PROPERTY;
 @property (readonly, nonatomic, strong) UISearchDisplayController* searchDisplayController;
 @property (readonly, nonatomic, strong) UIView* viewIfLoaded STUB_PROPERTY;
+@property (readonly, nonatomic, strong) UIView* modalOverlayView; // WinObjC extension
```

Comment:
If there's no objection, I'm going to push a commit in the coming days to remove this added WinObjC extension from the header. I'll just leave an _ prefixed method in the .mm so that clients can access the modalOverlayView via performSelector.